cmd/earth/main.go

```diff
@@ -454,20 +454,28 @@ func newEarthApp(ctx context.Context, console conslogging.ConsoleLogger) *earthA
 			Hidden: true,
 			Subcommands: []*cli.Command{
 				{
-					Name:   "create",
-					Action: app.actionOrgCreate,
+					Name:      "create",
+					Usage:     "create a new organization",
```

Comment:
Should use the same style as other usage fields: `Create a new organization`

Reply:
good call, fixed.